.github/workflows/.test.yml

```diff
@@ -97,7 +97,7 @@ jobs:
       -
         name: Create summary
         run: |
-          teststat -markdown $(find /tmp/reports -type f -name '*.json' -print0 | xargs -0) >> $GITHUB_STEP_SUMMARY
+          teststat -markdown $(find /tmp/reports -type f -name '*-go-test-report.json' -print0 | xargs -0) >> $GITHUB_STEP_SUMMARY
 
```

Comment:
Wondering if the actual mistake we have is to run that find in a subshell, and that swallowing errors. Do we need to split it up?

Reply:
To be clear; not against the fix as-is if it fixes the immediate issue, but while looking at this, I was wondering if we're tapering over the underlying issue (which, I think, may be an empty result, or incorrect match?) but maybe I'm entirely wrong there!

Reply:
Changed to `find ... -exec ... {} \+`